### PR TITLE
Added test cases TC01 and TC02

### PR DESCRIPTION
**Pull request by clemens29**

### CODE REVIEW

File: docs/test/test_cases.adoc [hunk 1]
```diff
@@ -1,14 +1,59 @@
 = Test Cases: {project-name}
-Vorname Nachname <email@domain.org>; Vorname2 Nachname2 <email2@domain.org>; Vorname3 Nachname3 <email3@domain.org>
+Clemens Näther <clemens.naether@stud.htw-dresden.de>;
 {localdatetime}
 include::../_includes/default-attributes.inc.adoc[]
 // Platzhalter für weitere Dokumenten-Attribute
 
 
 == Allgemeine Informationen
+Die nachfolgenden Test Cases dienen der Überprüfung und Sicherstellung der Funktionalität der entwickelten VM-Vergabe. Im Allgemeinen zielen Test Cases darauf ab, sicherzustellen, dass das zu testende Objekt korrekt funktioniert.
```
Comment (don-ny) [inline docs/test/test_cases.adoc:9]:
```suggestion
Die nachfolgenden Test Cases dienen der Überprüfung und Sicherstellung der Funktionalität des entwickelten Systems. Test Cases stellen sicher, dass das zu testende Objekt/der zu testende Anwendungsfall korrekt funktioniert.
```

**Review by don-ny — CHANGES_REQUESTED** [review]
- 1 suggestion for adoc
- please alter the .gitignore to prevent all __pycache__ folders from being committed (this PR should only contain 1 adoc file)

**Comment by clemens29** [comment]
> * 1 suggestion for adoc DONE
> * please alter the .gitignore to prevent all **pycache** folders from being committed (this PR should only contain 1 adoc file)

Actually I have the pycache in my .gitignore, the files are only changed because they got deleted in my branch but are still in the main-branch. Should I leave it like it is now or should I remove the pycache in my .gitignore so that there is only 1 file in this PR? @don-ny

**Review by don-ny — CHANGES_REQUESTED** [review]
I would like to have the __pycache__ files and folders removed from the project altogether. It's not really part of the test cases but could you do this please?

**Comment by clemens29** [comment]
Ok, I think the pycache folders should be deleted when we merge this to main

**Review by don-ny — APPROVED** [review]

**Pull request merged**